src/sec_newton.mbx

```diff
@@ -529,7 +529,7 @@
               <tabular>
                 <row>
                   <cell><m>x_0=</m></cell>
-                  <cell><m><var name="$x[0]" /></m></cell>
+                  <cell><var name="$x[0]" /></cell>
```

Comment:
This cell's contents should be in math mode. This is a var that displays a value as opposed to the ones that follow which are vars that server as answer blanks.